src/main/java/duke/Launcher.java

```diff
@@ -0,0 +1,18 @@
+package duke;
+
+import duke.fxml.Main;
+import javafx.application.Application;
```

Comment:
Consider standardizing import orders (e.g. java/javafx packages first then duke packages) in this and some other files. I like the nicely separately imports in Duke.java.